solution/ui/regulations/utilities/api.js

```diff
@@ -55,7 +55,12 @@ function configure(obj) {
     config = { ...config, ...obj };
 }
 
-function fetchJson(url, options = {}, retryCount = 0) {
+function fetchJson({
+    url,
+    options = {},
+    retryCount = 0,
+    cacheResponse = true,
```

Comment:
adding `cacheResponse` flag to base API fetch method.  If passed, API response is not set in localStorage and it is also will not get from localStorage.

Reply:
Also of note: we are now passing in a single object as the only function parameter.  No more worrying about parameter order when invoking this function.